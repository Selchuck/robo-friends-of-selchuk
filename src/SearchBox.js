import React from 'react';

const Searchbox = () => {
  return (
    <input type='search' placeholder='search robot'/>
    );
}

export default Searchbox;
