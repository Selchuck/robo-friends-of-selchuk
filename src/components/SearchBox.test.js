import { shallow } from 'enzyme';
import SearchBox from './SearchBox';
import React from 'react';

it('expect to render Searchbox component', () => {
    expect(shallow(<SearchBox />)).toMatchSnapshot();
});