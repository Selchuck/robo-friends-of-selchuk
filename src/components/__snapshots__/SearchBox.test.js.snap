// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render Scroll component 1`] = `ShallowWrapper {}`;

exports[`expect to render Searbox component 1`] = `ShallowWrapper {}`;

exports[`expect to render Searchbox component 1`] = `ShallowWrapper {}`;
